test/experimental_noalias_scope_decl.ll

```diff
@@ -0,0 +1,28 @@
+; RUN: llvm-as %s -o %t.bc
```

Comment:
The filename should be `test/llvm-intrinsics/experimental.noalias.scope.decl.ll` to align with other such tests